src/Compilers/CSharp/Portable/Compiler/MethodCompiler.cs

```diff
@@ -1803,6 +1803,8 @@ private static SyntaxToken GetImplicitConstructorBodyToken(CSharpSyntaxNode cont
             {
                 case SyntaxKind.ClassDeclaration:
                     return ((ClassDeclarationSyntax)containerNode).OpenBraceToken;
+                case SyntaxKind.InterfaceDeclaration:
+                    return ((InterfaceDeclarationSyntax)containerNode).OpenBraceToken;
```

Comment:
Maybe collapse all the cases and cast to `BaseTypeDeclarationSyntax` to get the OpenBrace ?